docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "docs.bruno",
+  "type": "collection"
+}
\ No newline at end of file
```

Comment:
It's a good practice to add a newline at the end of your files. This is a common convention that helps prevent any concatenation issues that might arise. [medium]